configs/Fedora/scoring.toml

```diff
@@ -0,0 +1,6 @@
+[Scoring]
+# This can set how bad each error is.
+# But we set BadnessThreshold to -1 so anything above 0
+# is actually fatal.
```

Comment:
This can be changed (and that's what we'll use in openSUSE):
`  -P, --permissive      treat individual errors as non-fatal`